Update readme (#189)
prometheus-erl/prometheus.erl: README.md

# Prometheus.io client for Erlang #

> [![Erlang/OTP Versions](https://img.shields.io/badge/erlang%2Fotp-26%7C27%7C28-blue)](https://www.erlang.org)

Reviewer: ```suggestion
[![Erlang/OTP Versions](https://img.shields.io/badge/erlang%2Fotp-26+-blue)](https://www.erlang.org)
```

Reviewer: We don't actually specify a max. and there's a change this'll work as is the day 29 comes out 😓

## Blogs

> - [Install, Monitor Erlang Releases in Kubernetes with Helm + Prometheus](https://spacetimeinsight.com/installing-monitoring-erlang-releases-kubernetes-helm-prometheus/)

Reviewer: Removed because of expired SSL certificate?

Author: Because it doesn't exist anymore 😅

## Compatibility

### Build tools

> [Rebar3](https://github.com/erlang/rebar3) is supported.

Reviewer: I'd actually use https://rebar3.org.

http_request_duration_milliseconds_sum{method="get"} 2622
```

> If you want to use a ready HTTP exporter, see the [Inets HTTPD Exporter](https://github.com/deadtrickster/prometheus_httpd) or the [Cowboy1/2 Exporters](https://hex.pm/packages/prometheus_cowboy).

Reviewer: ```suggestion
To use a ready-made HTTP exporter, check out the [Inets HTTPD Exporter](https://github.com/deadtrickster/prometheus_httpd) or the [Cowboy1/2 Exporters](https://hex.pm/packages/prometheus_cowboy).
```

All metrics created via `new/1` or `declare/1`. The difference is that `new/1` actually wants metric to be
new and raises `{mf_already_exists, {Registry, Name}, Message}` error if it isn't.

> Both `new/1` and `declare/1` accept options as [proplist](http://erlang.org/doc/man/proplists.html)

Reviewer: If we're using `ex_doc` `t:proplists:proplist` should work, right?
